Stripe/PublicHeaders/STPCustomerContext.h

```diff
@@ -35,7 +35,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param keyProvider   The key provider the customer context will use.
  @return the newly-instantiated customer context.
  */
-- (instancetype)initWithKeyProvider:(id<STPEphemeralKeyProvider>)keyProvider;
+- (instancetype)initWithKeyProvider:(id<STPCustomerEphemeralKeyProvider>)keyProvider;
```

Comment:
Renaming this class means:

1. We're also going to have to update the Examples, the stripe.com docs, and the [generated docs](https://stripe.github.io/stripe-ios/docs/index.html)
2. We'll need coordinate a release between all three.
3. Users will have to update their existing code when they upgrade, even if they're not using Issuing.

I don't think any of this is blockers, just thinking through the user-facing consequences here.

Reply:
@jack-stripe Just to make sure: is it in line with what you expected?

Reply:
I believe I safely deprecated `STPEphemeralKeyProvider` by changing it to subclass  `STPCustomerEphemeralKeyProvider` - this would trigger a deprecation warning in existing integrations but not a compiler error. Yes, this will mean a docs update though.